js/src/pages/price-benchmark/price-benchmark-suggestions.js

```diff
@@ -1,63 +1,120 @@
+		header: (
+			<Label labelKey={ LABEL_PRICE_CHANGE_EFFECTIVENESS } alignLeft />
+		),
+		label: LABELS[ LABEL_PRICE_CHANGE_EFFECTIVENESS ].title,
+		render: ( { item } ) => {
+			return (
+				<EffectivenessIndicator
+					effectiveness={ item[ 'price-change-effectiveness' ] }
+				/>
+			);
+		},
 	},
 	{
-		key: 'effectiveness',
-		label: __( 'Price Change Effectiveness', 'google-listings-and-ads' ),
-		isLeftAligned: true,
-		required: true,
+		id: 'regular-price',
+		enableHiding: false,
+		enableSorting: true,
+		enableGlobalSearch: false,
+		label: <Label labelKey={ LABEL_REGULAR_PRICE } />,
+		render: ( { item } ) => {
+			return <Price amount={ item[ 'regular-price' ] } highlight />;
+		},
 	},
 	{
-		key: 'regular-price',
-		label: __( 'Regular Price', 'google-listings-and-ads' ),
-		isLeftAligned: true,
-		required: true,
+		id: 'price-on-google',
+		enableHiding: false,
+		enableSorting: true,
+		enableGlobalSearch: false,
+		header: <Label labelKey={ LABEL_PRICE_ON_GOOGLE } />,
+		label: LABELS[ LABEL_PRICE_ON_GOOGLE ].title,
+		render: ( { item } ) => {
+			return <Price amount={ item[ 'price-on-google' ] } />;
+		},
 	},
 	{
-		key: 'avg-google-price',
-		label: __( 'Avg. Price on Google', 'google-listings-and-ads' ),
-		isLeftAligned: true,
-		required: true,
+		id: 'price-gap',
+		enableHiding: false,
+		enableSorting: true,
+		enableGlobalSearch: false,
+		header: <Label labelKey={ LABEL_PRICE_GAP } />,
+		label: LABELS[ LABEL_PRICE_GAP ].title,
+		render: ( { item } ) => {
+			return `${ item[ 'price-gap' ] }%`;
+		},
 	},
 	{
-		key: 'price-gap',
-		label: __( 'Price Gap', 'google-listings-and-ads' ),
-		isLeftAligned: true,
-		required: true,
+		id: 'suggested-price',
+		enableHiding: false,
+		enableSorting: true,
+		enableGlobalSearch: false,
+		header: <Label labelKey={ LABEL_SUGGESTED_PRICE } />,
+		label: LABELS[ LABEL_SUGGESTED_PRICE ].title,
+		render: ( { item } ) => {
+			return <Price amount={ item[ 'suggested-price' ] } />;
+		},
 	},
 	{
-		key: 'suggested-price',
-		label: __( 'Suggested Price', 'google-listings-and-ads' ),
-		isLeftAligned: true,
-		required: true,
-	},
-	{
-		key: 'actions',
-		label: __( 'Action', 'google-listings-and-ads' ),
-		isLeftAligned: true,
-		required: true,
+		id: 'action',
+		enableHiding: false,
+		enableSorting: false,
+		enableGlobalSearch: false,
+		label: LABELS[ LABEL_ACTION ].title,
+		render: ( { item } ) => {
+			return <ChangePrice id={ item.id } />;
+		},
 	},
 ];
 
+const TABLE_FIELDS_MOBILE = [ 'action' ];
+
+/**
+ * PriceBenchmarkSuggestions component.
+ *
+ * This component fetches and displays price benchmark suggestions using the
+ * `usePriceBenchmarkSuggestions` hook. It renders a table with the suggestions
+ * data and handles responsiveness by providing different field configurations
+ * for desktop and mobile views.
+ *
+ * @return {JSX.Element} A div containing the PriceBenchmarkTable component.
+ */
 const PriceBenchmarkSuggestions = () => {
```

Comment:
Will this component also render the MetricsNotice?

Reply:
I originally had it like that @joemcgill :
https://github.com/woocommerce/google-listings-and-ads/pull/2853/commits/c6020d5173e9ada06db62ad856e7c59a017b29a1
But then i thought the `PriceBenchmarkTable` should only contain fields that should be part of the basic table. Then any components using `PriceBenchmarkTable` should pass their extra fields. I feel that can be more flexible if we have different table types in the future. Let me know what you think. Happy to go with your suggestion as well 😁

Reply:
Is there a use case you have in mind where another component would need to use `PriceBenchmarkTable`? Seems like any custom tables that need different data could do so and wrap their own DataViews component.

Reply:
So eventually, we’ll have the `PriceBenchmarkAdjustments` component. My main concern is that `PriceBenchmarkTable` will contain logic related to both suggestions and adjustments, even though we already have (or will have) dedicated components like `PriceBenchmarkSuggestions` and `PriceBenchmarkAdjustments` that encapsulate that logic. In other words, I’d prefer not to split some of logic across both `PriceBenchmarkTable` and `PriceBenchmarkSuggestions`. What do you think?